store/test-store/tests/chain/ethereum/manifest.rs

```diff
@@ -187,6 +187,58 @@ specVersion: 0.0.2
     assert_eq!(12345, graft.block);
 }
 
+#[tokio::test]
+async fn parse_indexer_hints() {
+    const YAML: &str = "
+dataSources: []
+schema:
+  file:
+    /: /ipfs/Qmschema
+graft:
+  base: Qmbase
+  block: 12345
+specVersion: 0.0.2
+indexerHints:
+  prune:
+    historyBlocks: 100
+";
+
+    let manifest = resolve_manifest(YAML, SPEC_VERSION_0_0_4).await;
+
+    assert_eq!("Qmmanifest", manifest.id.as_str());
+    assert_eq!(
+        manifest
+            .indexer_hints
+            .unwrap()
+            .prune
+            .unwrap()
+            .history_blocks
+            .unwrap(),
+        100
+    );
+}
+
+#[tokio::test]
+async fn parse_indexer_hints_prune() {
+    const YAML: &str = "
+dataSources: []
+schema:
+  file:
+    /: /ipfs/Qmschema
+graft:
+  base: Qmbase
+  block: 12345
+specVersion: 0.0.2
+indexerHints:
+  prune:
+";
+
+    let manifest = resolve_manifest(YAML, SPEC_VERSION_0_0_4).await;
+
+    assert_eq!("Qmmanifest", manifest.id.as_str());
+    assert!(manifest.indexer_hints.unwrap().prune.is_some());
```

Comment:
I don't understand this test - what is this trying to ensure?

Reply:
Oh i used that test to test out the indexer_hints parsing is working properly.

Reply:
Will remove it.

Reply:
its was removed already. merging

Reply:
Just a comment what the intent of the test was would have been fine, too - it was just a bit confusing.